ccd-definition/AuthorisationCaseField/CareSupervision/judiciary.json

```diff
@@ -1315,6 +1315,13 @@
     "UserRole": "caseworker-publiclaw-judiciary",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "languageRequirement",
```

Comment:
is this casefield used anywhere outside of event? doesn't look like it? if not then don't think some of these permissions are needed (i.e case field permissions can just match event permissions)

Reply:
Aligned the case field permissions to the event permissions